### DISCUSSION TITLE
reading/2023/010-%E5%B0%8F%E7%8E%8B%E5%AD%90

### DISCUSSION THREAD

**Discussion by giscus** [GuoXiCheng/guoxicheng.github.io#192] [Announcements]
# reading/2023/010-%E5%B0%8F%E7%8E%8B%E5%AD%90

《小王子》 （法）圣埃克絮佩里 只有用心才能看见。本质的东西用眼是看不见的。 正是你为你的玫瑰花费的时光，才使你的玫瑰变得如此重要。 

https://guoxicheng.top/reading/2023/010-%E5%B0%8F%E7%8E%8B%E5%AD%90.html

**Comment by Paper-Dragon** [comment]
所有的大人都曾经的小孩，虽然，只有少数人记得。
